astropy/time/formats.py

```diff
@@ -1753,4 +1778,6 @@ def _broadcast_writeable(jd1, jd2):
     return s_jd1, s_jd2
 
 
+# Import symbols from core.py that are used in this module. This succeeds
+# because __init__.py imports format.py just before core.py.
```

Comment:
Yes thanks!!! It is also important for `core` itself, since that imports `format` too.